sdks/go/test/integration/integration.go

```diff
@@ -27,7 +27,8 @@
 // Running integration tests can be done with a go test call with any flags that
 // are required by the test pipelines, such as --runner or --endpoint.
 // Example:
-//    go test -v ./sdks/go/test/integration/... --runner=portable --endpoint=localhost:8099
+//
```

Comment:
```suggestion
```

Reply:
Interestingly, `go fmt` added this line, so I'm actually inclined to leave it. It is kinda odd though

Reply:
oh okay then